### PR TITLE
Clarifying

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1971

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/collaboration/public-folders/batch-migration-from-previous-versions.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ For detailed syntax and parameter information, see the following topics:
 
    - _Maximum mailbox size in bytes_ equals the maximum size you want to set for the new public folder mailboxes. When specifying this setting, be sure to allow for expansion so the public folder mailbox has room to grow.
 
-   - _Folder to size map path_ equals the file path of the .csv file you created when running the `Export-PublicFolderStatistics.ps1` script.
+   - _Folder to size map path_ equals to the full file path of the .csv file you created when running the `Export-PublicFolderStatistics.ps1` script.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/collaboration/public-folders/batch-migration-from-previous-versions.md:245]:
```suggestion
   - _Folder to size map path_ equals the full file path of the .csv file you created when running the `Export-PublicFolderStatistics.ps1` script.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**